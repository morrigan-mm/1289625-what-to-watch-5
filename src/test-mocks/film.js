const film = {
  id: 15,
  poster: `#poster`,
  preview: `#preview`,
  video: `#video`,
  cover: `#cover`,
  title: `The Revenant`,
  genre: `Action`,
  releaseDate: 2015,
  description: `A frontiersman on a fur trading`,
  rating: 4,
  ratingCount: 618498,
  director: `The Director`,
  actors: [`actor1`, `actor2`],
  runtime: 156,
  addedToMyList: false,
  backgroundColor: `#92918B`,
  backgroundImage: `#bgimage`
};

export default film;
