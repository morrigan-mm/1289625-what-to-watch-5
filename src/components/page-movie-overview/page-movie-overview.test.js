import React from "react";
import renderer from "react-test-renderer";
import film from "../../test-mocks/film";
import PageMovieOverview from "./page-movie-overview";


describe(`PageMovieOverview`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(
          <PageMovieOverview film={film} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
