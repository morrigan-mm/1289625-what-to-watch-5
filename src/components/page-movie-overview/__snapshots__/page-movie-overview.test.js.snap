// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageMovieOverview Should render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      4,0
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Normal
      </span>
      <span
        className="movie-rating__count"
      >
        618498
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      A frontiersman on a fur trading
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        The Director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        actor1, actor2
         and other
      </strong>
    </p>
  </div>,
]
`;
